fluentd-daemonset/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fluent/fluentd:v1.2.1-debian-onbuild
+MAINTAINER Petr Kotas <petr.kotas@gmail.com>
+
+RUN buildDeps="sudo make gcc g++ libc-dev ruby-dev" \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends $buildDeps \
+ && sudo gem install \
+        fluent-plugin-secure-forward \
+        fluent-plugin-record-reformer \
+        fluent-plugin-kubernetes_metadata_filter \
+        ffi \
```

Comment:
This is installed in all images in the fluentd repo which I checked. E.g.: https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/docker-image/v1.2/debian-syslog/Gemfile